setup.py

```diff
@@ -137,8 +137,9 @@ def run_tests(self):
         'show': 'matplotlib',
         'fftw': 'pyfftw',
         'pywavelets': 'Pywavelets',
-        'scikit' : 'scikit-image',
-        'all': test_requires + ['matplotlib', 'pyfftw', 'Pywavelets', 'scikit-image']
+        'scikit': 'scikit-image',
+        'dicom': 'dicom',
```

Comment:
Other way around. To the left of the colon is our name for it, to the right is the pip package name.